src/aspire/basis/fpswf_2d.py

```diff
@@ -177,8 +177,8 @@ def _generate_pswf_quad(
             ]
         )
 
-        pts_x = quad_rule_pts_r * np.cos(quad_rule_pts_theta)
-        pts_y = quad_rule_pts_r * np.sin(quad_rule_pts_theta)
+        pts_x = quad_rule_pts_r * np.sin(quad_rule_pts_theta)
```

Comment:
May want to rename here for clarity (x should still be radius times cosine, no?). Important thing to make sure is that coordinates are returned in the order `pts_y, pts_x`.

Reply:
Right, that would be less confusing. Easy change. Thanks.